<!DOCTYPE html>
<html lang='en'>

<head>
    <base href=".">
    <link rel="shortcut icon" type="image/png" href="pictures/logos/smbe_favicon.jpg"/>
    <link rel="stylesheet" type="text/css" media="all" href="main.css"/>
    <meta name="description" content="SMBE Satellite Meeting Biological Noise">
    <meta name="resource-type" content="document">
    <meta name="distribution" content="global">
    <meta name="KeyWords" content="Conference">
    <title>SMBE Satellite Meeting Biological Noise</title>
</head>

<body>
        <style>
.date {
        width: 150px; /* Adjust this value as needed */
        text-align: left; /* Optional: Ensures content is aligned */
}
        </style>
        <div class="banner">
                <img src="pictures/tuebingen_1.jpg" alt="Conference Banner" class="banner-image">
                <div class="banner-text">
                        <p style="font-size: 1.1em;">SMBE Satellite Meeting on</p>
                        <p>Origin, Maintenance, and Evolution of</p>
                        <h1>Biological Noise</h1>
                </div>
        </div>
                <table class="navigation">
                        <tr>
                                <td class="navigation">
                                        <a title="Conference Home Page" href="index.html">Home</a>
                                </td>
                                <td class="navigation">
                                        <a title="Register for the Conference" href="registration.html">Registration</a>
                                </td>
                                <td class="navigation">
                                        <a class="current" title="Conference Program" href="program.html">Program</a> 
                                </td>
                                <td class="navigation">
                                        <a title="Location of the Conference" href="location.html">Location</a>
                                </td>
                                <td class="navigation">
                                        <a title="Conference Flyer" href="flyer.html">Flyer</a>
                                </td>
                        </tr>
                </table>



                <h2 id="day0">Day 0: Monday 2nd June </h2>
                <table>
                        <tr>
                                <td class="date" rowspan="4">
                                        18:00 - 
                                </td>
                                <td class="title">
                                        Cocktail Hour at Max Planck Institute for Biology
                                </td>
                        </tr>
                </table>
                <!-- Day 1 -->
                <!-- am -->

                <h2 id="day1">Day 1: Tuesday 3rd June </h2>
                <table>
                        <tr>
                                <td class="date" rowspan="4">
                                        08:45 - 09:00
                                </td>
                                <td class="title">
                                        Welcome
                                </td>
                        </tr>
                </table>

                <table>
                        <tr>
                                <td class="date" rowspan="4">
                                        09:00 - 10:00
                                </td>
                                <td class="title">
                                        Plenary talk: Stochasticity in Biology: a Philosophical Analysis of its Potential Explanatory Role
                                </td>
                        </tr>
                        <tr>
                                <td class="speaker">
                                        Francesca Merlin
                                </td>
                        </tr>
                        <tr>
                                <td class="abstract">
                                        After briefly reviewing the history of the concept of noise in biological research, I will discuss a number of philosophical issues, both conceptual and epistemological, about the way biological noise is invoked and studied. Then, in line with recent theoretical and experimental results, especially in cellular and developmental biology, but not only, I will argue for noise, which I will rather call "stochasticity", as an explanatory concept in biology. In particular, I will address the question of what stochasticity (i.e., chance more generally) can explain in this disciplinary context and how.

                                </td>
                        </tr>
                </table>

                <table>
                        <tr>
                                <td class="date" rowspan="4">
                                        10:00 - 10:20
                                </td>
                                <td class="title">
                                        Understanding gene expression “by chance”
                                </td>
                        </tr>
                        <tr>
                                <td class="speaker">
                                        Marco Casali
                                </td>
                        </tr>
                        <tr>
                                <td class="abstract">
                                        In the 2000s, theoretical and experimental research on noise in biology focused on its effects on gene expression (Elowitz et al 2002, Klingenberg 2005, Samoilov et al 2006, Maheshri & O’Shea 2007, Raj & Oudernaarden 2008, Pipel 2011). In this context, the term “noise” generally refers to various random microscopic events that occur inside the cell and produce fluctuations in one or more steps of this intracellular process (e.g. transcription and translation). Biologists are increasingly pointing out that random noise can play a role in the development of biological systems, sometimes even a functional one, rather than always being a mere nuisance (e.g, Eldar and Elowitz, 2010; Feinberg and Irizarry, 2010; Gupta et al. 2011; Losick and Desplan 2008; Maamar et al. 2007; Mettetal and van Oudenaarden 2007; Meyer and Roeder 2014; Pilpel 2011; Pujadas and Feinberg 2012; Raser and O’Shea 2005; Raj and van Oudenaarden 2008). I take this recent research seriously and argue that chance, which affects various biological processes involved in cell function and organismal development, may actually play a positive, constructive role and should therefore be understood as more than just noise (see also Huang, 2009). More specifically, the aim of this talk is to highlight that the notion of chance in molecular biology can have original epistemic value in terms of understanding. The proposal is to argue that the notion of chance acts as an abstractor that enables the agent to understand - through synthesis and visualization - the process in question more deeply. This proposal will be summarized by developing the notion of chance as an abstractor (CA) and analyzing examples from molecular biology. It is shown how this CA account finds its roots in biological practice, and some of the advantages of CA are discussed, as well as an objection. Key words: chance, stochasticity, gene expression, molecular biology, abstraction, understanding.
                                </td>
                        </tr>
                </table>
                <table>
                        <tr>
                                <td class="date" rowspan="1">
                                        10:20 - 10:50
                                </td>
                                <td class="title-special">
                                        Coffee break
                                </td>
                        </tr>
                </table>

                <h3> Session 1: Non-genetic inheritance of phenotypic noise</h3>
                Chair: Ignacio Bravo
                <br><br>
                In this session, we will consider how phenotypic noise can be inherited through non-genetic
                mechanisms. We will explore the various epigenetic, environmental, and stochastic factors that
                contribute to the transmission of phenotypic variability across generations. <br><br>

                <table>
                        <tr>
                                <td class="date" rowspan="4">
                                        10:50 - 11:50
                                </td>
                                <td class="title">
                                        Keynote: From Splicing Noise to Disease: When Errors Matter
                                </td>
                        </tr>
                        <tr>
                                <td class="speaker">
                                        Maria Carmo-Fonseca
                                </td>
                        </tr>
                        <tr>
                                <td class="abstract">
                                        Splicing is inherently noisy, with errors occurring occasionally during splice site recognition. Yet, these errors are normally tolerated in healthy individuals. Genetic variation may enhance splicing noise, without disrupting cellular homeostasis. In my laboratory, we are interested in understanding the interplay between stochastic splicing errors and those driven by genetic variants, focusing on the thresholds that separate benign variability from pathogenic consequences. By dissecting splicing errors linked to genetic variants that are not associated with disease, I will discuss how cells buffer splicing noise and when this tolerance fails, leading to disease phenotypes. Understanding these thresholds may provide key insights into the evolutionary pressures shaping transcriptome resilience and the molecular origins of genetic disorders.
                                </td>
                        </tr>
                </table>
                <table>
                        <tr>
                                <td class="date" rowspan="4">
                                        11:50 - 12:10
                                </td>
                                <td class="title">
                                        Epigenetic switching as an adaptive strategy: harnessing biochemical noise and other evolutionary trade-offs
                                </td>
                        </tr>
                        <tr>
                                <td class="speaker">
                                        Mariana Gómez-Schiavon
                                </td>
                        </tr>
                        <tr>
                                <td class="abstract">
                                        Biological noise, inherent in all cellular processes, plays a fundamental role in shaping evolutionary dynamics. Organisms must navigate both environmental fluctuations and stochasticity in gene expression, yet the evolutionary forces governing these interactions remain unresolved. One key strategy is adaptive variation, where organisms generate phenotypic diversity within and between individuals. A particularly intriguing mechanism is epigenetic switching—bistable molecular systems built from self-reinforcing feedback loops that allow spontaneous, heritable phenotype switching in the absence of genetic mutations. These epimutations, driven by biochemical noise, enable rapid adaptation to environmental changes but impose a cost in stable conditions due to frequent, unnecessary transitions. Using in silico evolution of mechanistic gene regulatory circuit models, we investigate how epigenetic switching emerges as an adaptive strategy in fluctuating environments. We identify a fundamental trade-off between minimizing adaptation time and maintaining phenotypic robustness, both shaped by the ultrasensitivity of gene regulatory circuits. This trade-off stems from the interplay between biochemical noise and genotype structure, with bistable epigenetic switches evolving preferentially in fast-changing environments. Through a lineage-tracing approach, we distinguish three adaptation strategies—genetic adaptation, epigenetic switching, and a hybrid bistable adaptation, where mutations fine-tune switching rates. Control simulations confirm that epigenetic switches evolve specifically to harness biochemical noise, rather than as accidental byproducts of selection. Our findings underscore the role of biological noise in shaping evolutionary strategies, revealing how noise is not merely a constraint but a key driver of adaptive complexity.
                                </td>
                        </tr>
                </table>
                <table>
                        <tr>
                                <td class="date" rowspan="4">
                                        12:10 - 12:30
                                </td>
                                <td class="title">
                                        Beyond the Egg: Hidden Impact of Maternal Contribution Variability
                                </td>
                        </tr>
                        <tr>
                                <td class="speaker">
                                        Anastasiia Berezenko
                                </td>
                        </tr>
                        <tr>
                                <td class="abstract">
                                        The maternal contribution encompasses factors present in the egg that influence the development, phenotype, and survival of the offspring. Among the various factors, RNA plays a crucial role in many species, including vertebrates. Maternally provided RNA orchestrates the early development before the transition from maternal to zygotic control, known as the maternal-to-zygotic transition (MZT). Importantly, maternal RNA was confirmed to be a variability source in various species including Ciona intestinalis, Neogobius melanostomus, Drosophila melanogaster, etc. We postulate that this variability plays a pivotal role in the evolution of populations and their adaptation to rapidly changing environmental conditions and is influenced by genetic and environmental factors with the aim of diversification in stressful conditions. We therefore aimed to determine the extent of variability in maternally provided RNAs in zebrafish under optimal versus stressful conditions. In our study, we investigate the effect of heatshock in mothers at different timepoints before egglay on the maternal RNA load through single-egg sequencing. Our approach allows us to differentiate between-egg and within-clutch variability from between-mother variability and between-treatment variability on a global scale and on a gene-by-gene level. We hope that this will shed new light on the role of heritable, non-genetic variability in the face of environmental change and deepen our understanding of its role in evolutionary biology and adaptation.
                                </td>
                        </tr>
                </table>


                <table>
                        <tr>
                                <td class="date" rowspan="1">
                                        12:30 - 14:00
                                </td>
                                <td class="title-special">
                                        Lunch
                                </td>
                        </tr>
                </table>

                <!-- Day 1 -->
                <!-- pm  -->

                <table>
                        <tr>
                                <td class="date" rowspan="1">
                                        14:00 - 15:00
                                </td>
                                <td class="title">
                                        Poster session 1
                                </td>
                        </tr>
                </table>
                <table>
                        <tr>
                                <td class="date" rowspan="1">
                                        15:00 - 16:00
                                </td>
                                <td class="title">
                                        Discussion
                                </td>
                        </tr>
                </table>
                <table>
                        <tr>
                                <td class="date" rowspan="2">
                                        16:00 - 
                                </td>
                                <td class="title-special">
                                        Social activity & dinner
                                </td>
                        </tr>
                        <tr>
                                <td class="abstract">
                                        Stocherkahn boat ride along the Neckar river
                                </td>
                        </tr>
                </table>



                <!-- Day 2 -->
                <!-- am -->

                <h2>Day 2: Wednesday 4th June </h2>

                <h3> Session 2: The genetic basis of phenotypic noise</h3>
                Chair: Julien Dutheil
                <br><br>
                In this session, we investigate the distinct genetic sources of phenotypic noise. We will study how noise
                propagates through distinct levels (transcription, translation, regulatory networks, cells, tissues) from
                genomes to organisms.
                <table>
                        <tr>
                                <td class="date" rowspan="4">
                                        09:00 - 10:00
                                </td>
                                <td class="title">
                                        Keynote: Evolution of Regulatory Networks Controlling Plasticity in Gene Expression between <i>Saccharomyces cerevisiae</i> and <i>Saccharomyces paradoxus</i>
                                </td>
                        </tr>
                        <tr>
                                <td class="speaker">
                                        Patricia Wittkopp
                                </td>
                        </tr>
                        <tr>
                                <td class="abstract">
                                        Organisms often cope with changes in their environments by modifying gene expression levels, which can affect their cellular function. This plasticity in gene expression arises when cells sense a change in their environment and alter the activity or availability of <i>trans</i>-regulatory factors, which interact with each gene’s <i>cis</i>-regulatory sequences to determine its expression. To understand how regulatory networks controlling plasticity in gene expression evolve, we used RNAseq data from the baker’s yeast <i>Saccharomyces cerevisiae</i>, its close relative <i>Saccharomyces paradoxus</i>, and their F1 hybrids collected at multiple time points following the transfer of cells from standard laboratory conditions to four different environments (low phosphorus, low nitrogen, increased temperature, and decreased temperature). We also looked at how gene expression changes throughout the cell cycle and during the transition from log phase to stationary phase. In all six datasets, we compared gene expression levels between <i>S. cerevisiae</i> and <i>S. paradoxus</i> and asked how expression plasticity has diverged. We then tested for divergence in expression plasticity between the two species-specific alleles in F1 hybrids, allowing us to disentangle the effects of divergence in <i>cis</i>- and <i>trans</i>-regulation. We found many cases where expression plasticity had diverged between species, most of which were unique to a single environment and attributable primarily to trans-regulatory divergence. We then investigated potential sources of this <i>trans</i>-regulatory divergence by testing 46 transcription factors for effects on expression plasticity resulting from cells being moved to the low nitrogen environment. We found several cases where deletion of a transcription factor had different effects on a gene's expression plasticity in the two species, implying divergence in the regulatory network controlling environment-specific expression. With prior work identifying such changes for only a few genes between much more distantly related species, this work suggests that connections in regulatory networks are changing more often, and over shorter evolutionary timescales, than previously appreciated.
                                </td>
                        </tr>
                </table>

                <table>
                        <tr>
                                <td class="date" rowspan="1">
                                        10:00 - 10:30
                                </td>
                                <td class="title-special">
                                        Coffee break
                                </td>
                        </tr>
                </table>

                <table>
                        <tr>
                                <td class="date" rowspan="4">
                                        10:30 - 10:50
                                </td>
                                <td class="title">
                                        The rate and molecular mechanisms underlying the evolution of gene expression noise
                                </td>
                        </tr>
                        <tr>
                                <td class="speaker">
                                        Brian Metzger
                                </td>
                        </tr>
                        <tr>
                                <td class="abstract">
                                        Development and survival are dependent on the correct level and timing of gene expression. However, gene expression is stochastic and thus varies among genetically identical cells even in the same environment. This ‘noise’ in expression varies substantially among genes and can be acted on by natural selection. However, the extent of differences in noise among species, the rate at which expression noise evolves, and the molecular mechanisms that contribute to changes in expression noise are typically unknown. To address these questions, we performed single-cell RNA-seq on two distantly related Saccharomyces yeast species co-cultured in a common environment. Unlike the widespread changes in gene expression observed between these species, gene expression noise was similar for the majority of genes. This suggests a level of independent genetic control between gene expression and expression noise, and that expression noise is either under greater selective constraint or is a more robust phenotype that has a lower mutational input than gene expression. To identify the molecular basis of expression noise differences between these species, we additionally performed single-cell RNA-seq on an F1 hybrid between these species and used allele-specific differences to estimate expression noise of both species genomes in a common cellular environment. We found that few differences in expression noise existed within the hybrids, indicating that most differences in expression noise between these species are trans-acting. This was again in contrast to gene expression, where many genes had cis-regulatory differences between these species. Finally, expression within individual hybrid cells was poorly correlated between alleles, consistent with independent, and thus intrinsic, control of expression noise. Together, these results suggest that gene expression noise evolves via different molecular and evolutionary mechanisms than gene expression. The extent to which these differences constrain or bias the direction and types of evolutionary changes in gene expression remains to be determined.
                                </td>
                        </tr>
                </table>
                <table>
                        <tr>
                                <td class="date" rowspan="4">
                                        10:50 - 11:10
                                </td>
                                <td class="title">
                                        Trans-acting mutations reveal genetic modulators of intrinsic and extrinsic gene expression noise
                                </td>
                        </tr>
                        <tr>
                                <td class="speaker">
                                        Fabien Duveau
                                </td>
                        </tr>
                        <tr>
                                <td class="abstract">
                                        Gene expression regulation is a stochastic process that can be modified by mutations not only in a deterministic manner but also in a probabilistic way: previous studies from us and others reported cases where mutations changed the extent of cell-to-cell variability in gene expression (also called “gene expression noise”) without necessarily changing the expected (mean) expression level. Such mutations can act either in cis (perturbing expression noise of gene at their own locus) or in trans (perturbing expression noise of a gene located elsewhere in the genome). Although systematic studies have successfully analyzed the properties of cis-acting mutations modulating gene expression noise, much less is known about trans-acting modulators. In particular, what type of random mutations in the genome may affect expression noise of a given gene is unclear. Here we used yeast strains generated by random mutagenesis to map trans-acting modulators of the expression noise of a reporter gene regulated by the yeast TDH3 promoter (pTDH3-YFP). Using a bulked segregant analysis followed by whole-genome sequencing (BSA-seq) specifically designed to map mutations affecting expression noise, we identified three mutations that we validated individually by allele replacement and flow-cytometry. These mutations target the coding sequences of genes that were not expected to participate to noise control. Using a dual reporter system and other genetic tools, we showed that each mutation affected either extrinsic noise (variability due to cell-specific factors), intrinsic noise (inherent variability within each cell), or both. Our results show that diverse and often unpredictable genetic mechanisms may contribute to the control of cell-to-cell variability in gene expression.
                                </td>
                        </tr>
                </table>

                <table>
                        <tr>
                                <td class="date" rowspan="4">
                                        11:10 - 11:30
                                </td>
                                <td class="title">
                                        The costs and benefits of altered translation accuracy during bacterial adaptation to antibiotics
                                </td>
                        </tr>
                        <tr>
                                <td class="speaker">
                                        Laasya Samhita
                                </td>
                        </tr>
                        <tr>
                                <td class="abstract">
                                        Protein synthesis, while central to the functioning of cells, is also highly error-prone. Mistranslation is generally costly, but beneficial in specific contexts, including during growth in antibiotics. However, the cost of changing translation accuracy relative to the fitness cost imposed by external selection pressures, and its evolutionary compensation, remains unknown. We quantified the cost of genetically increasing and decreasing mistranslation rates in E. coli, and found that it was generally lower than the fitness cost imposed by low concentrations of several antibiotics. The antibiotic cost was quickly compensated by all strains during experimental evolution, via both antibiotic- and genotype- specific mutations. In contrast, accuracy costs were compensated only in some cases in the presence of antibiotics, without clear causal mutations. Interestingly, control populations that evolved without antibiotics consistently compensated the cost of accuracy, and also evolved increased antibiotic resistance. Our work demonstrates that although the cost of altered translation accuracy is generally weak, it can shape both adaptive outcomes and underlying genetic mechanisms, with strong collateral fitness effects on apparently unrelated phenotypes such as antibiotic resistance.
                                </td>
                        </tr>
                </table>

                <table>
                        <tr>
                                <td class="date" rowspan="1">
                                        11:30 - 12:30
                                </td>
                                <td class="title">
                                        Poster session 2
                                </td>
                        </tr>
                </table>
                <table>
                        <tr>
                                <td class="date" rowspan="1">
                                        12:30 - 14:00
                                </td>
                                <td class="title-special">
                                        Lunch
                                </td>
                        </tr>
                </table>

                <!-- Day 2-->
                <!-- pm -->

                <table>
                        <tr>
                                <td class="date" rowspan="1">
                                        14:30 - 15:00
                                </td>
                                <td class="title">
                                        Discussion
                                </td>
                        </tr>
                </table>


                <h3> Session 3: The organismal implications of phenotypic noise</h3>
                Chair: Luisa Pallares
                <br><br>
                The degree of phenotypic noise is modulated by environmental conditions (e.g., ecological or
                developmental) in a trait-dependent manner. In this session, we will address how such environmental
                effects on phenotypic noise might influence the adaptation process.
                <table>
                        <tr>
                                <td class="date" rowspan="4">
                                        15:00 - 16:00
                                </td>
                                <td class="title">
                                        Keynote: Drift in Individual Behavioral Phenotype as a Strategy for Unpredictable Worlds
                                </td>
                        </tr>
                        <tr>
                                <td class="speaker">
                                        Benjamin de Bivort
                                </td>
                        </tr>
                        <tr>
                                <td class="abstract">
                                        Individuals, even with matched genetics and environment, show substantial phenotypic variability. This variability may be part of a bet-hedging strategy, where populations express a range of phenotypes to ensure survival in unpredictable environments. In addition phenotypic variability between individuals (“bet-hedging”), individuals also show variability in their phenotype across time, even absent external cues. There are few evolutionary theories that explain random shifts in phenotype across an animal's life, which we term drift in individual phenotype. We use individuality in locomotor handedness in Drosophila melanogaster to characterize both bet-hedging and drift. We use a continuous circling assay to show that handedness spontaneously changes over timescales ranging from seconds to the lifespan of a fly. We compare the amount of drift and bet-hedging across a number of different fly strains and show independent strain specific differences in bet-hedging and drift. We show manipulation of serotonin changes the rate of drift, indicating a potential circuit substrate controlling drift. We then develop a theoretical framework for assessing the adaptive value of drift, demonstrating that drift may be adaptive for populations subject to selection pressures that fluctuate on timescales similar to the lifespan of an animal. We apply our model to real world environmental signals and find patterns of fluctuations that favor random drift in behavioral phenotype, suggesting that drift may be adaptive under some real world conditions. These results demonstrate that drift plays a role in driving variability in a population and may serve an adaptive role distinct from population level bet-hedging.
                                </td>
                        </tr>
                </table>

                <table>
                        <tr>
                                <td class="date" rowspan="1">
                                        16:00 - 16:30
                                </td>
                                <td class="title-special">
                                        Coffee break
                                </td>
                        </tr>
                </table>

                <table>
                        <tr>
                                <td class="date" rowspan="4">
                                        16:30 - 16:50
                                </td>
                                <td class="title">
                                        Phenotypic plasticity, epigenetics and the inheritance of non-genetic information in a predatory nematode
                                </td>
                        </tr>
                        <tr>
                                <td class="speaker">
                                        Ralf J. Sommer
                                </td>
                        </tr>
                        <tr>
                                <td class="abstract">
                                        Developmental plasticity and polyphenisms, the ability of a single genotype to form multiple phenotypes in response to environmental variation, have been proposed to represent a major facilitator of evolutionary divergence and novelty. While still contentious after a long time of neglect, the role of plasticity is gaining popular support through the development of various model systems that provide molecular and epigenetic insight into associated processes. We will review the current understanding of mouth-form plasticity in the nematode Pristionchus pacificus that exhibits a mouth dimorphism with the eurystomatous (Eu) form being a potential predator on nematodes, whereas the stenostomatous (St) form is a strict bacterial feeder. Mouth-form dimorphism represents an example of a bi-stable epigenetic switch resulting in an ecologically relevant bet-hedging strategy. Work on the genetic and epigenetic regulation of P. pacificus mouth-form plasticity for more than a decade revealed the existence of a major developmental switch gene that is central for i) genetic regulation, ii) sensing of environmental variation resulting in epigenetic control, and iii) surprisingly, also functions as a mutational hotspot that creates massive natural variation. This epigenetic switch gene is part of a complex gene-regulatory-network, which contains genes with fundamentally different evolutionary histories. Recently performed long-term environmental induction experiments of 110 genetically identical lines for 101 generations with dietary switching found immediate and systemic diet-induced plasticity, resulting exclusively in the formation of the predatory Eu morph. Strikingly, periodic diet-reversals revealed transgenerational memory that entails multigenerational plasticity. Integrative studies on P. pacificus mouth-form development combine these molecular investigations that aim at an understanding of the regulation of polyphenisms with field work and modelling to study the evolutionary significance and ecological relevance of plasticity. Finally, we will review ideas about developmental plasticity itself originating from molecular and developmental noise.
                                </td>
                        </tr>
                </table>
                <table>
                        <tr>
                                <td class="date" rowspan="4">
                                        16:50 - 17:10
                                </td>
                                <td class="title">
                                        Establishment, maintenance and consequences of inter-individual transcriptional variability in plants
                                </td>
                        </tr>
                        <tr>
                                <td class="speaker">
                                        Sandra Cortijo
                                </td>
                        </tr>
                        <tr>
                                <td class="abstract">
                                        Surprisingly, differences in phenotypes and gene expression are observed between genetically identical individuals grown in the same environment. While we now have a good knowledge of the source and consequences of transcriptional differences observed between cells, in particular for unicellular organisms, it is still very scarce when it comes to variability between multicellular organisms. Using plants as a model we analysed the establishment, maintenance and consequences of inter-individual transcriptional variability. We showed, for a gene of interest, that differences in expression between plants are established in young seedlings, maintained over several days but not transmitted to the next generation. Our results also indicate that these differences in expression can explain phenotypic variability between plants such as for the root growth. Finally, using a genome-wide approach, we found a co-expression in seedlings for our gene of interest, involved in nitrate nutrition, and genes involved in photosynthesis. All in all, our study suggests that a global coordination of the genes involved in the carbon/nitrate balance in plants is established in young seedlings, with differences of this state between plants, and then maintained over time.
                                </td>
                        </tr>
                </table>

                <table>
                        <tr>
                                <td class="date" rowspan="4">
                                        17:10 - 17:30
                                </td>
                                <td class="title">
                                        Sex-Specific Effects of Developmental Temperature on Dorsal Cluster Neuron Variability, Wiring Asymmetry, and Behavior
                                </td>
                        </tr>
                        <tr>
                                <td class="speaker">
                                        Mohammad Amine Reslan
                                </td>
                        </tr>
                        <tr>
                                <td class="abstract">
                                        Sexual dimorphisms in insects are evident in various morphological traits and behaviors, such as oviposition and courtship song production. These behaviors are typically governed by sex-specific neural circuits. In Drosophila melanogaster, a subset of visual interneurons known as Dorsal Cluster Neurons (DCNs or LC14 neurons) play a crucial role in object orientation response in both sexes. DCNs exhibit substantial variation in cell body and axon counts and innervate both the lobula and medulla neuropils of the fly visual system. Notably, asymmetry in the axonal projections of medulla-innervating DCNs (M-DCNs) correlates with increased visual fixation in both males and females. However, males, on average, exhibit a higher degree of asymmetry and consequently stronger visual fixation. To investigate the effect of developmental temperature on DCN anatomy and its associated behavior, we raised flies at 12°C, 18°C, 25°C (optimal), and 29°C, covering the viable temperature range for D. melanogaster. We used the UAS-Gal4 system to visualize DCN anatomy and Buridan’s assay to assess visual fixation behavior. Our results show that axon count scales with developmental temperature, with lower temperatures significantly increasing axon number in both sexes. Females consistently exhibited higher axon counts at 18°C, 25°C, and 29°C. Absolute asymmetry in axon number between hemispheres remained stable across sexes and temperatures. Behaviorally, flies raised at 18°C showed a similar visual fixation profile to those raised at 25°C but exhibited reduced locomotor activity. However, females responded differently than males. At 29°C, visual fixation was significantly reduced, likely due to increased locomotor activity leading to decreased fixation accuracy. These findings suggest that DCN development is influenced by stochastic processes, as indicated by stable within-individual and among-individual variation in hemispheric asymmetry. While environmental factors affect neural architecture similarly in both sexes, behavioral outcomes differ, suggesting an underlying genetic component linked to the sex-determination pathway. We are currently testing how these genetic mechanisms shape sexually dimorphic visual behaviors.
                                </td>
                        </tr>
                </table>

                <table>
                        <tr>
                                <td class="date" rowspan="1">
                                        16:30 - 19:00
                                </td>
                                <td class="title-special">
                                        Free time
                                </td>
                        </tr>
                </table>
                <table>
                        <tr>
                                <td class="date" rowspan="1">
                                        19:00 - 
                                </td>
                                <td class="title-special">
                                       Dinner 
                                </td>
                        </tr>
                </table>

                <!-- Day 3 -->
                <!-- am -->

                <h2>Day 3: Thursday 5th June </h2>

                <h3> Session 3: The organismal implications of phenotypic noise (Continued)</h3>

                <table>
                        <tr>
                                <td class="date" rowspan="1">
                                        09:00 - 10:00
                                </td>
                                <td class="title">
                                        Poster session 3
                                </td>
                        </tr>
                </table>
                <table>
                        <tr>
                                <td class="date" rowspan="1">
                                        10:00 - 10:30
                                </td>
                                <td class="title-special">
                                        Coffee break
                                </td>
                        </tr>
                </table>
                <table>
                        <tr>
                                <td class="date" rowspan="1">
                                        10:30 - 11:30
                                </td>
                                <td class="title">
                                        Discussion
                                </td>
                        </tr>
                </table>


                <h3> Session 4: Population genetics of phenotypic noise </h3>
                Chair: Daniel Weinreich
                <br><br>
                By definition, biological noise is random, but its statistical structure can be influenced by an organism’s
                genetics. This raises the question of how natural selection regards genetic variants that influence the
                statistical structure of biological noise, and that will be the topic of this session.
                <table>
                        <tr>
                                <td class="date" rowspan="4">
                                        11:30 - 12:30
                                </td>
                                <td class="title">
                                        Keynote: The Population Genetics of Biological Noise
                                </td>
                        </tr>
                        <tr>
                                <td class="speaker">
                                        Daniel Weinreich
                                </td>
                        </tr>
                        <tr>
                                <td class="abstract">
                                        Noise is intrinsic to information transmission, and information transmission is intrinsic to life. Critically, while biological noise is random, its amount in any organism is in part genetically determined. Here, we explore the population genetic fate of alleles that heritably influence the amount of biological noise in their carrier. We argue from first principles that biological noise is a double-edged sword: almost always deleterious but also occasionally yielding high fitness phenotypes. We further suggest that this implies the existence of an equilibrium amount of noise, at which the advantage of producing additional, rare beneficial phenotypes is exactly balanced by the cost of producing the vastly more common deleterious phenotypes. We predict that the location of this equilibrium in any species will reflect the rate at which its environment is changing, the heritability of realized noise, the model of selection, and population size. Our framework generalizes modifier theory sensu Altenberg, Feldman and others, and resolves teleological criticisms of the hypothesis that evolvability can evolve. We conclude with an outline of open theoretical and empirical questions posed by this framework. 
                                </td>
                        </tr>
                </table>

                <table>
                        <tr>
                                <td class="date" rowspan="1">
                                        12:30
                                </td>
                                <td class="title-special">
                                        Group picture
                                </td>
                        </tr>
                </table>
                <table>
                        <tr>
                                <td class="date" rowspan="1">
                                        12:30 - 14:00
                                </td>
                                <td class="title-special">
                                        Lunch
                                </td>
                        </tr>
                </table>

                <!-- Day 3 -->
                <!-- pm -->

                <table>
                        <tr>
                                <td class="date" rowspan="4">
                                        14:00 - 14:20
                                </td>
                                <td class="title">
                                        Proliferation variability and fate of mutators generated de novo by optogenetics
                                </td>
                        </tr>
                        <tr>
                                <td class="speaker">
                                        Gael Yvert
                                </td>
                        </tr>
                        <tr>
                                <td class="abstract">
                                        Appearance of mutators in a population of cells or individuals represents a special situation during which biological noise suddenly increases. In this case, a lineage acquires a globally-high mutation rate, often due to a defect in DNA replication or repair processes, and the resulting increase in random mutations can inflate inter-individual phenotypic variation in this lineage as compared to their “wild-type” counterparts. The fate of such a lineage is then driven by a trade-off during selection: more frequent mutations provide a larger substrate for adaptation and selection (“accelerated innovation”), but these mutations can also impair physiology (“mutational burden”). The recurrent detection of mutators in various contexts, such as cancer, drug resistance or experimental evolution, suggests that the benefit of accelerated innovation may sometimes dominate the cost of mutational burden. Yet, whether and how mutators contribute to natural selection is a long-standing question because their phenotypes and dynamics are difficult to characterize. To study mutators appearing in a population of cells, we developed a novel experimental approach based on optogenetics and encapsulation of single cells in micro-reactors. This method can be applied to monitor phenotypic changes resulting from any specific genetic alteration of interest. Here, it enabled us to observe populations where de novo mutators were purged or fixed after they were generated by light. We experimentally tracked over time the proliferation rate of microcolonies seeded by individual mutator cells, providing a dynamic view of the evolution of their fitness distributions. Our preliminary results reveal subtle differences in the statistical properties of mutational burden across environmental conditions. We are now studying populations where mutators can grow under neutral or positive selection.
                                </td>
                        </tr>
                </table>
                <table>
                        <tr>
                                <td class="date" rowspan="4">
                                        14:20 - 14:40
                                </td>
                                <td class="title">
                                        A simple null model captures key features of non-deterministic genotype-phenotype maps
                                </td>
                        </tr>
                        <tr>
                                <td class="speaker">
                                        Nora Martin
                                </td>
                        </tr>
                        <tr>
                                <td class="abstract">
                                        Genotype-phenotype (GP) maps are important for evolutionary processes, since they translate genotypic mutations into phenotypic variation. Such GP maps can be modelled computationally for molecular phenotypes, for example RNA secondary structures, but GP map concepts are also relevant for more macroscopic phenotypes. Thus, there is a variety of GP maps describing different systems and phenotypes, and a general framework needs to be established: a set of quantities to characterise and compare GP maps, and to reveal features relevant for evolutionary processes. Such a unified framework has primarily been developed for simple ‘deterministic’ maps, where each genotype corresponds to a single categorical phenotype. However, biological systems are often more complex: For example, in the RNA model, a sequence does not fold into a single structure, but is better approximated by a Boltzmann ensemble of different structures. This is captured by more complex non-deterministic/plastic GP maps, where each genotype generates a probability distribution of phenotypes. Recently, quantities characterising such plastic maps have been proposed, but they have only been applied to a small number of maps and many open questions about their origin and implications for evolving populations remain. In this contribution, I will analyse three biophysical examples of such non-deterministic maps, and put the results into context with a simple non-biological null model. This model constitutes a simple and flexible basis for future simulations of evolving populations on such maps. Thus, this research will contribute to a theoretical basis that can be applied to further non-deterministic maps.
                                </td>
                        </tr>
                </table>

                <table>
                        <tr>
                                <td class="date" rowspan="4">
                                        14:40 - 15:00
                                </td>
                                <td class="title">
                                        Optimal division asymmetry and fitness effects of noisy transcription and growth in <i>E. coli</i>
                                </td>
                        </tr>
                        <tr>
                                <td class="speaker">
                                        Ulrich Steiner 
                                </td>
                        </tr>
                        <tr>
                                <td class="abstract">
                                        Most bacterial cells reproduce by binary fission, but contrasting classical assumptions, the resulting offspring are not identical. My interest lies in whether this heterogeneity among isogenic individuals has evolved to an optimum, and how it might be maintained and selected on. First, I ask what level of size and transcription signal asymmetry optimizes population growth, i.e. fitness. Second, I ask how does noise in growth, transcription signal dynamics, and asymmetry influences population growth rates. To answer these questions, I use high throughput single-cell bacteria data on transcription signal dynamics, cell growth and division. With these data I advance and parameterize structured population models to scale within individual dynamics and division asymmetries to population dynamics. My findings show that the observed size and transcription signal asymmetry has evolved to an optimum, as increasing or decreasing division asymmetry lowers population growth. For effects of cell intrinsic noise, cell growth and transcription signal noise does not influence population growth. For division asymmetry noise, population growth is lowered only with reduced noise—reduced variance—in transcription signal asymmetry, but not growth asymmetry or the covariance among the two. It is interesting that the between cell asymmetry after division can already be detected within regions of the cell before division. My findings also suggest a deep organismal integration, likely governed by trade-offs, between transcription dynamics and growth. I will end discussing implications for aging and phenotypic antibiotic resistance.
                                </td>
                        </tr>
                </table>
                <table>
                        <tr>
                                <td class="date" rowspan="1">
                                        15:00 - 16:00
                                </td>
                                <td class="title">
                                        Poster session 4
                                </td>
                        </tr>
                </table>

                <table>
                        <tr>
                                <td class="date" rowspan="1">
                                        16:00 - 16:30
                                </td>
                                <td class="title-special">
                                        Coffee break
                                </td>
                        </tr>
                </table>


                <table>
                        <tr>
                                <td class="date" rowspan="1">
                                        16:30 - 17:30
                                </td>
                                <td class="title">
                                        Discussion
                                </td>
                        </tr>
                </table>
                <table>
                        <tr>
                                <td class="date" rowspan="1">
                                        17:30 - 19:00
                                </td>
                                <td class="title">
                                        Final discussion & wrap up
                                </td>
                        </tr>
                </table>
                <table>
                        <tr>
                                <td class="date" rowspan="1">
                                        19:00 - 
                                </td>
                                <td class="title-special">
                                        Conference dinner
                                </td>
                        </tr>
                </table>


                <h2>Day 4: Friday 6th June </h2>
                Departure




                <footer>
                        &copy; Conference Organizers
                        &nbsp;|&nbsp; Original design by <a href="https://github.com/mikepierce">Mike Pierce</a>
                </footer>

        </body>
</html>

